src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -62,17 +62,59 @@ Rules generally consist of three parts:
 - _MAC Addresses_
 - _Anything_: Matches any value. This is useful if you want to remove a certain JSON key by path using [_Selectors_](#selectors) regardless of the value.
 
+{% capture __alert_content -%}
+
+Sentry does not know if a local variable that looks like a creditcard number actually is one. As such you need to expect false-positives but also false-negatives. [_Selectors_](#selectors) can help you in limiting the scope in which your rule runs.
+
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Sentry does not know what your code does"
+  content=__alert_content
```

Comment:
```suggestion
  content=__alert_content
  level="warning"
```

Not sure about this one, but I've seen and used warning for notes like this. Here's the difference rendered:

<img width="946" alt="image" src="https://user-images.githubusercontent.com/88819/77178590-8fb10280-6ac7-11ea-9467-b73ffd689317.png">